src/OrbitGl/ThreadTrack.cpp

```diff
@@ -457,12 +457,12 @@ void ThreadTrack::OnCaptureComplete() {
     return current_time + 1;
   }
 
-  // Making a ceiling with integers to not lose precision. Given a track width of 4000 pixels, we
-  // can capture for 53 days without overflowing.
-  int next_pixel = ((ns_from_min * num_pixels_on_track) + total_ns - 1) / total_ns;
+  // Given a track width of 4000 pixels, we can capture for 53 days without overflowing.
+  int current_pixel = (ns_from_min * num_pixels_on_track) / total_ns;
```

Comment:
You had `ns_from_min * num_pixels_on_track - 1` before, but now `-1` is gone.
Is this intended?

Reply:
Yes, it's intended. We changed the logic there because it was wrong when we were exactly in the boundary. Now should be easy to understand.